configs/config.ci.yaml

```diff
@@ -5,48 +5,49 @@ logging:
   # level: debug
   # Set to false to get a json output for the log
   console: true
+  location: true
 
 web:
   port: 8000
 
 database:
   host: localhost
   port: 5432
-  user: postgres
-  password: postgres
-  name: postgres
+  user: idmsvc-user
+  password: idmsvc-secret
+  name: idmsvc-db
 
-kafka:
-  auto:
-    offset:
-      reset: latest
-    commit:
-      interval:
-        ms: 5000
-  bootstrap:
-    servers: localhost:9092
-  group:
-    id: idmsvc
-  message:
-    send:
-      max:
-        retries: 15
-  request:
-    timeout:
-      ms: 30000
-    required:
-      acks: -1
-  retry:
-    backoff:
-      ms: 100
-  timeout: 10000
-  topics:
-    - platform.idmsvc.introspect
-  # sasl:
-  #   username: someusername
-  #   passowrd: somepassword
-  #   mechanism: somemechanism
-  #   protocol: someprotocol
+# kafka:
```

Comment:
Please remove the block instead of commenting it out. If we need it later, we can pull it from git history.